packages/react-components/review/beta/react-components.api.md

```diff
@@ -1343,6 +1343,12 @@ export interface ImageGalleryStrings {
     downloadButtonLabel: string;
 }
 
+// @beta
+export interface InlineImage {
```

Comment:
Just thinking about this a bit more, would it also be useful to return the ChatAttachment object in here? So instead of needing to do the look up again, they have all the information handy?

Reply:
export declare interface ChatAttachment {
    /** Id of the attachment */
    id: string;
    /** The type of attachment. */
    attachmentType: AttachmentType;
    /** The type of content of the attachment, if available */
    contentType?: string;
    /** The name of the attachment content. */
    name?: string;
    /** The URL where the attachment can be downloaded */
    url: string;
    /** The URL where the preview of attachment can be downloaded */
    previewUrl?: string;
}

ChatAttachment and imgAttrs will overlap for many props such as id, url name, etc.
And ChatAttachment is missing some potential properties that imgAttrs may have, such as style.
If you think certain props of ChatAttachment is useful to contoso, maybe we can add those to the imgAttrs when we call the onRenderInlineImage callback?

Reply:
I'm just thinking if there is a nicer way to send the `url` to the developer at the same time.